http/src/main/resources/reference.conf

```diff
@@ -1228,7 +1228,7 @@ autoFreeze {
 }
 
 autodelete {
```

Comment:
it would be great to have a small comment here that says the current minimum autodelete threshold is 1hr